src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -17,6 +17,14 @@ public Category save(Category category) {
         return categoryRepository.save(category);
     }
 
+    @Override
+    public Category update(Long id, Category category) {
+        if (categoryRepository.existsById(id)) {
```

Comment:
Since throwing an exception is not a regular program flow, I'd revert these `if`s in the update methods and put the exception inside the `if` body, but that's a minor thing